modules/Bio/EnsEMBL/Compara/PipeConfig/PairAligner_conf.pm

```diff
@@ -545,12 +545,19 @@ sub core_pipeline_analyses {
               -can_be_empty  => 1,
               -rc_name => $self->o('filter_duplicates_himem_rc_name'),
            },
- 	   {  -logic_name => 'update_max_alignment_length_after_net',
- 	      -module     => 'Bio::EnsEMBL::Compara::RunnableDB::GenomicAlignBlock::UpdateMaxAlignmentLength',
-	      -rc_name => '1Gb_job',
-	      -wait_for =>  [ 'create_filter_duplicates_net_jobs', 'filter_duplicates_net', 'filter_duplicates_net_himem' ],
+           {  -logic_name    => 'remove_partial_blocks',
+              -module        => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+              -parameters    => { 
+                                'sql' => "DELETE FROM genomic_align_block WHERE genomic_align_block_id NOT IN (SELECT genomic_align_block_id FROM genomic_align)"
+                                },
+              -flow_into     => [ 'run_healthchecks' ],
+              -wait_for      => [ 'create_filter_duplicates_net_jobs', 'filter_duplicates_net', 'filter_duplicates_net_himem' ],
+           },
```

Comment:
This new runnable should be added in its corresponding place in the pipeline, i.e. before `run_healthchecks`. Having it here has required me some extra time to realise what you where doing.